src/mfa.js

```diff
@@ -25,7 +25,7 @@ export async function sendSms({ type, phoneNumber, firstFactorCode } = {}) {
         );
       }
 
-      return sendVerificationCode({
+      return sendMfaCode({
```

Comment:
Not sure about changing the name of this as the guide uses "verification code": https://userfront.com/docs/api-client.html#log-in-with-verification-code